rust/Cargo.toml

```diff
@@ -21,6 +21,7 @@ async-trait = "0.1"
 bytes = "1"
 chrono = { version = "0.4.22", default-features = false, features = ["clock"] }
 cfg-if = "1"
+dashmap = "5"
```

Comment:
This is only needed for Datafusion, so can it be made optional

Reply:
I initially planned to also implement our trait. but then also thought we should maybe slightly update ours .. but yes, this can (and should) right now be optional.